packages/ui/src/components/Radio/radio.tsx

```diff
@@ -26,15 +26,16 @@ const Radio: IRadio = (props) => {
     const { label, icon, labelProps, disabled, onChange, children, id, ...rest } = props
 
     const handleChange = useCallback((event: RadioChangeEvent) => {
-        const stringContent = label ? label : extractChildrenContent(children)
+        const stringContent = extractChildrenContent(children)
```

Comment:
Why are you extracting it not from label?

Reply:
And it's probably will broke analytics, since it contains dynamic postfix 🤔

Reply:
I left it like before.
Essentially no difference